src/maker/api.rs

```diff
@@ -300,10 +296,6 @@ impl Maker {
             config.tor_auth_password = tor_auth_password;
         }
 
-        if matches!(connection_type, Some(ConnectionType::TOR)) {
-            check_tor_status(config.control_port, config.tor_auth_password.as_str())?;
-        }
```

Comment:
We still need to check Tor status. Instead of matching with the connection type, it should be matched with not(feature=integration-test).